### PR TITLE
Optimized autoload of classes fails (#92)

### PR DESCRIPTION
**Pull request by sunaoka**
Optimized autoload of classes fails in composer version 2.

This has not been fixed in v0.9.1, so send a pull request.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing](../../paypayopa-sdk-php/blob/master/contributing.md) document?
* [x] Have you read and signed the automated Contributor's License Agreement?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../paypayopa-sdk-php/pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Review by ashraf-km — APPROVED** [review]
Approved

File: tests/PayloadsTest.php [hunk 1]
```diff
@@ -6,12 +6,15 @@
...
 use PayPay\OpenPaymentAPI\Models\CreatePaymentAuthPayload;
 use PayPay\OpenPaymentAPI\Models\CreatePaymentPayload;
+use PayPay\OpenPaymentAPI\Models\CreatePendingPaymentPayload;
 use PayPay\OpenPaymentAPI\Models\CreateQrCodePayload;
 use PayPay\OpenPaymentAPI\Models\ModelException;
 use PayPay\OpenPaymentAPI\Models\OrderItem;
 use PayPay\OpenPaymentAPI\Models\RefundPaymentPayload;
 use PayPay\OpenPaymentAPI\Models\RevertAuthPayload;
 
+use function PayPay\OpenPaymentAPI\Helpers\GetRand;
```
Comment (shreyanshp) [inline tests/PayloadsTest.php:16]: @sunaoka Why is this required?

Comment (sunaoka) [reply tests/PayloadsTest.php:16]: @shreyanshp This is because the GetRand() function is used in the testModelFailures() method (line 274).

**Review by shreyanshp — APPROVED** [review]

**Comment by shreyanshp** [comment]
@sunaoka Thank you for the PR, it is merged  and released as 0.9.2
https://github.com/paypay/paypayopa-sdk-php/releases/tag/0.9.2
